flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/JarHandlerParameterTest.java

```diff
@@ -211,6 +218,37 @@ void testProvideJobId() throws Exception {
         assertThat(jobGraph.get().getJobID()).isEqualTo(jobId);
     }
 
+    @Test
+    void testProvideFlinkConfig() throws Exception {
+
+        REQB jarRequestBodyWithFlinkConfig = getJarRequestBodyWithFlinkConfig(FLINK_CONFIGURATION);
+        HandlerRequest<REQB> request =
+                createRequest(
+                        getJarRequestBodyWithFlinkConfig(FLINK_CONFIGURATION),
```

Comment:
```suggestion
                        jarRequestBodyWithFlinkConfig,
```

Reply:
The method name and getJarRequestBodyWithJobId to maintain consistent style